llvm/lib/Transforms/Instrumentation/HWAddressSanitizer.cpp

```diff
@@ -1935,12 +1940,12 @@ void HWAddressSanitizer::ShadowMapping::init(Triple &TargetTriple,
     SetFixed(0);
   } else if (ClMappingOffset.getNumOccurrences() > 0) {
     SetFixed(ClMappingOffset);
-    WithFrameRecord = false;
   } else if (ClEnableKhwasan || InstrumentWithCalls) {
     SetFixed(0);
     WithFrameRecord = false;
```

Comment:
It is still implied from `InstrumentWithCalls`. Why?

Reply:
We change it for `ClMappingOffset`. Please be more explicit in the commit message about what behaior changes.

Reply:
No, that's exactly what I mean.
We will not set 'WithFrameRecord = false;' from ClMappingOffset or ClMappingOffsetDynamic

Reply:
Please add

"Do not disable frame records for fixed offset"

to the commit message, because that's what this change also does, correct?